jobs/queue-processor/src/main.py

```diff
@@ -25,9 +25,12 @@
 DB_HOST = os.getenv("POSTGRES_SERVER", default="psql-rds.default")
 DB_NAME = os.getenv("POSTGRES_DB", default="curibio")
 
-# TODO figure out how to pass these
-MANTARRAY_LOGS_BUCKET_ENV = kclient.V1EnvVar(name="MANTARRAY_LOGS_BUCKET_ENV", value="prod-mantarray-logs")
-UPLOADS_BUCKET_ENV = kclient.V1EnvVar(name="UPLOADS_BUCKET_ENV", value="prod-pulse3d-uploads")
+MANTARRAY_LOGS_BUCKET = kclient.V1EnvVar(
+    name="MANTARRAY_LOGS_BUCKET_ENV", value=os.getenv("mantarray_logs_bucket")
+)
+PULSE3D_UPLOADS_BUCKET = kclient.V1EnvVar(
+    name="UPLOADS_BUCKET_ENV", value=os.getenv("pulse3d_uploads_bucket")
+)
```

Comment:
@tannermpeterson it would just mean these would be capitalized in `os.getenv`

Reply:
I think to be consistent let's capitalize

Reply:
*we decided not to since the other CRDs do not